frontend/src/pages/ResetPassword.vue

```diff
@@ -104,7 +104,11 @@ export default {
         })
         .catch((error) => {
           this.toastError(error.message)
-          if (error.message.includes('Code is older than 10 minutes'))
+          if (
```

Comment:
This is terrible code. Furthermore, the previous code did not match any message sent by the server. :(